build.gradle

```
@@ -40,6 +47,13 @@ buildscript {
             // Jetpack
             "preference"                  : "androidx.preference:preference-ktx:1.1.1",
 
+            // Networking
+            "retrofit"                    : "com.squareup.retrofit2:retrofit:${versions.retrofit}",
+            "gson"                        : "com.squareup.retrofit2:converter-gson:${versions.gson}",
+
+            // Dependency Injection
+            "koin"                        : "io.insert-koin:koin-android:${versions.koin}",
```

We should be safe declaring the version numbers here within the `ext.deps` array since these version numbers are only referenced once. I think we have the pattern of adding values to `ext.versions` when that specific library version is referenced more than once.

```suggestion
// Networking
            "retrofit"                    : "com.squareup.retrofit2:retrofit:2.9.0",
            "gson"                        : "com.squareup.retrofit2:converter-gson:2.3.0",

            // Dependency Injection
            "koin"                        : "io.insert-koin:koin-android:3.1.2",
```